dmosopt/dmosopt.py

```diff
@@ -204,9 +204,9 @@ def _reduce_evals(self):
         self.x = self.x[perm[0 : self.population_size], :]
         self.y = self.y[perm[0 : self.population_size], :]
```

Comment:
How about x and y, are they also assumed to be one-dimensional?

Reply:
x, y, and c are all two-dimensional, where the second dimension is the number of variables (x), number of objectives (y) or number of constraints (c). But I think the colon is actually redundant in this case.

Reply:
Alright, let me test it.